### PR TITLE
add first alesco meeting minutes

### PR DESCRIPTION
**Pull request by jonathanspw**

### CODE REVIEW

**Review by codyro — APPROVED** [review]

**Review by Conan-Kudo — APPROVED** [review]

**Review by LKHN — APPROVED** [review]

File: docs/alesco/meeting-minutes.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# ALESCo Meeting Minutes
+- [2024-07-24](/alesco/meeting-minutes/2024-07-24)
```
Comment (bennyvasquez) [inline docs/alesco/meeting-minutes.md:2]:
```suggestion
---
Title: 'ALESCo Meeting Minutes'
---

Each meeting of ALESCo is public, and future meetings can be found on [events.almalinux.org](https://events.almalinux.org/category/7/)

# ALESCo Meeting Minutes
- [July 24, 2024](/alesco/meeting-minutes/2024-07-24)
```

**Review by bennyvasquez — COMMENTED** [review]
Adding a title and more context for the page to make it look less bare. Also linked to where future meeting details can be found. Also changed the date displayed so it's easier for folks in countries that swap the month and day in their date displays.

**Comment by codyro** [comment]
Once @bathomas accepts the org invite, please add him as a reviewer @jonathanspw

**Comment by jonathanspw** [comment]
> Once @bathomas accepts the org invite, please add him as a reviewer @jonathanspw

Yeah I couldn't get the tag to match him.  I tried.

**Review by Conan-Kudo — APPROVED** [review]

**Review by andrewlukoshko — APPROVED** [review]

**Review by bennyvasquez — APPROVED** [review]

**Pull request merged**